shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-suite/src/test/java/org/apache/shardingsphere/test/integration/engine/ddl/BaseDDLIT.java

```diff
@@ -139,7 +139,8 @@ private List<DataSetColumn> getActualColumns(final Connection connection, final
             while (resultSet.next()) {
                 DataSetColumn each = new DataSetColumn();
                 each.setName(resultSet.getString("COLUMN_NAME"));
-                each.setType(resultSet.getString("TYPE_NAME").toLowerCase());
+                String typeName = resultSet.getString("TYPE_NAME");
+                each.setType("CHARACTER VARYING".equals(typeName) ? "varchar" : typeName.toLowerCase());
```

Comment:
- I think the behavior in the context of `BaseDDLIT#assertTableMetaData()` is very strange. 

- `DataSetMetaData expected = getDataSet().findMetaData(tableName);` for this function, get `shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-suite/src/test/resources/cases/ddl/dataset/db/create_table.xml` similar file with `DataSetLoader#getFile()`, get the expected `TYPE_NAME`. 

- This is compared to the `TYPE_NAME` that the `BaseDDLIT#getActualColumns` method gets from `getActualDataSourceMap().get(each.getDataSourceName()).getConnection()`, the expected `VARCHAR` is always `CHARACTER VARYING`. 

- A minor problem is that `java.sql.Types` does not have `CHARACTER VARYING`, I don't know what's going on.

- I don't Know if this is a legacy issue, after all so many IT modules only need to change this file.

Reply:
Since https://www.h2database.com/html/datatypes.html does not exist for VARCHAR, I realize that this class may have to be changed.

Reply:
Refer to https://github.com/h2database/h2database/issues/3359#issuecomment-1009780245.